src/DependencyInjection/Compiler/FormFactoryCompilerPass.php

```diff
@@ -12,18 +12,30 @@
 namespace Sonata\CoreBundle\DependencyInjection\Compiler;
 
 use Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler\FormPass;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 
 /**
  * @deprecated since 3.7, to be removed in 4.0, the form mapping feature should be disabled.
  */
-class FormFactoryCompilerPass extends FormPass
+class FormFactoryCompilerPass implements CompilerPassInterface
```

Comment:
I think this is an acceptable BC-break

Reply:
Not sure if this is a BC-break: FormPass implements CompilerPassInterface. Isn't it allowed in this case?

Reply:
It's a BC-break for code that does `instanceof FormPass`. So, acceptable, allowed, whatever adjective floats your boat.